quickstarts/dotnet/blazor-server/config.yml

```diff
@@ -0,0 +1,48 @@
+slug: blazor-server
+
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Server
+  Microsoft developed Blazor Server to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Server integration with the .NET agent lets you monitor your apps with our quickstart dashboards right out of the box.
+
+  ## Why monitor Blazor Server?
+  Blazor Server offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Server allows you to run your code through Blazor Server on the server or the client, and you can decide which users or devices experience each method.
+  
+  There are a few compelling arguments that support the employment of Blazor Server:
+  - All of the major browsers support Blazor Server.
+  - Using C# for interactive web apps.
+  - Utilize current C# libraries.
+  - Near-native performance.
+  - Debugging and tooling.
+
+  ## What’s included in this quickstart?
+  Get these Blazor Server monitoring features out of the box with our quickstart:
+  - Understand the health of your system with dashboards that show throughput and error rate data, logs, web transaction time, Apdex score, and more. 
+  - Alerts.
+  - Monitor scripts and functions.
+  - Monitor web transactions.
+summary: | 
+  Instrument and monitor your Blazor Server application with our .NET agent. Analyze your performance data right out of the box.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Blazor Server
+documentation:
+  - name: Blazor Server installation docs
+    description: |
+      Our .NET agent can be used to instrument and monitor your Blazor Server application. Perform an immediate analysis of your performance statistics. 
```

Comment:
"Our .NET APM agent can be used to instrument and monitor your Blazor Server application. Perform an immediate analysis of your performance statistics."